packages/components/src/components/Modal/Modal.test.tsx

```diff
@@ -43,43 +49,14 @@ describe("<Modal />", () => {
     const renderedOpenButton = screen.getByText(OPEN_MODAL_TEXT);
     expect(renderedOpenButton).toBeInTheDocument();
 
-    await userEvent.click(renderedOpenButton);
+    await user.click(renderedOpenButton);
     expect(screen.getByRole("dialog")).toBeInTheDocument();
     expect(screen.getByRole("heading", { level: 2 })).toBeInTheDocument();
 
     const renderedSecondaryButton = screen.getByText("Back");
     expect(renderedSecondaryButton).toBeInTheDocument();
 
-    await userEvent.click(renderedSecondaryButton);
-    expect(screen.queryByRole("dialog")).not.toBeInTheDocument();
-  });
-
-  it("should open a modal and close it using the escape key", async () => {
```

Comment:
is this being removed because it doesn't make sense anymore?

Reply:
Because these scenarios are already being tested by Ariakit, as we inherit both behaviors when utilizing their component, in my perspective.
But let me know what you think. 🙏🏻